runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOreOverlay.java

```diff
@@ -66,59 +72,85 @@ public Dimension render(Graphics2D graphics)
 
 		// If no ores have even been collected, don't bother showing anything
 		if (nuggetsFound == 0 && coalFound == 0 && goldFound == 0 && mithrilFound == 0
-			&& adamantiteFound == 0 && runiteFound == 0)
+			|| adamantiteFound == 0 && runiteFound == 0)
```

Comment:
This is still wrong.
```suggestion
			&& adamantiteFound == 0 && runiteFound == 0)
```